incubator/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/streams/network/unsubscribe.publish.as.one.packet/client.rpt

```diff
@@ -0,0 +1,64 @@
+#
```

Comment:
Why is this scenario called `unsubscribe.publish.as.one.packet`?
I don't see any `publish` in the network protocol script below.
Also, check "packet" terminology per earlier comment.

Reply:
There is one, it's just easy to miss :)
`       [0x30 0x14]                   # PUBLISH
       [0x00 0x0a] "sensor/two"      # topic name
       [0x00]                        # properties
       "message"                     # payload`

Reply:
Ah yes, got it. 👍 

Perhaps would be clearer to read the script if you add a blank line before the `PUBLISH` packet, as it won't affect the script behavior if you don't add another `write` keyword.